easybuild/easyconfigs/g/gperftools/gperftools-2.14-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'gperftools'
+version = '2.14'
```

Comment:
Hmm, we're still at 2.13 for GCCcore/13.2.0 and 2.12 for GCCcore/12.3.0, should we really use 2.14 for 12.2.0

Reply:
Why would we not use latest `gperftools` version if it's not used yet in a particular generation of easyconfigs?

Reply:
For the sake of monotonically increasing versions :-)